packages/block-editor/src/components/iframe/content.scss

```diff
@@ -22,8 +22,11 @@
 }
 
 .block-editor-iframe__html {
+	$beizer: cubic-bezier(0.65, 0, 0.45, 1);
+	$duration: 0.5s;
+	border: 0 solid $gray-300;
 	transform-origin: top center;
-	transition: transform 0.3s;
+	transition: transform $duration $beizer, background $duration $beizer, border $duration $beizer, margin $duration $beizer;
```

Comment:
Would `all` be simpler?

Reply:
Yea, that works. I wanted to keep concerns separate but I tested it.